sdk/brcodePayment/rule/rule.js

```diff
@@ -0,0 +1,24 @@
+const SubResource = require('../../utils/subResource').SubResource
+
+
+class Rule extends SubResource {
+    /**
+     *
+     * BrcodePayment.Rule object
+     *
+     * @description The BrcodePayment.Rule object modify the behavior of BrcodePayment objects created in your Workspace.
```

Comment:
Melhor deixar essa docstring para o objeto rules: `The BrcodePayment.Rule object modifies the behavior of BrcodePayment objects when passed as an argument upon their creation.`. Que é a mesma que está em Python e Java